packages/components/psammead-episode-list/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.0.25 | [PR#4425](https://github.com/bbc/psammead/pull/4425) This package is being deprecated |
```

Comment:
What's the thinking behind saying it's _being_ deprecated here? Is there going to be a follow-up PR?

Reply:
Nope!

```suggestion
| 1.0.25 | [PR#4425](https://github.com/bbc/psammead/pull/4425) This package has been deprecated |
```